assignments/this.js

```diff
@@ -1,10 +1,10 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Window/Global Object Binding. To me, this means that "this" is always going to look outward to find meaning. It will keep searching until it hits the global object.
+* 2. Implicit binding: "This" referes to whatever is preceding the DOT. It is implied.
+* 3. New Binding: the "new" keyword creates a new object from a constructor function. "This" refers to the new object being created.
+* 4. Explicit binding: Whenever you us a method like .call, there explicit instructions given to what "this" refers to. I think it works as long as the new object is already created. Otherwise"this" does not know what it's refering to. 
```

Comment:
> I think it works as long as the new object is already created. Otherwise"this" does not know what it's refering to. 

:question: Can you elaborate a little more on this line?